### ISSUE TITLE
Complete icon in transcript button not announced

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-media#337]
### Subject of the issue
When using a screen reader, the icon to indicate a transcript has been read is not announced when this button receives focus.

<img width="316" height="144" alt="Image" src="https://github.com/user-attachments/assets/ea9742bf-3343-414b-b2ee-cb7e3ef5246b" />

A graphical symbol alone cannot be used to convey information, as this would exclude users who cannot perceive or understand that symbol. 

### Expected behaviour
A 'completed' label should read for consistency with other completed items/buttons e.g. "Completed. Transcript" or "Completed. Close Transcript".
